arch/arm/boot/dts/bcm2835-rpi.dtsi

```diff
@@ -27,6 +27,13 @@
 			firmware = <&firmware>;
 			#power-domain-cells = <1>;
 		};
+
```

Comment:
This is an upstream file - please don't modify it, except by trying to push changes upstream.

Downstream DT has a framebuffer node in bcm2708-rpi.dtsi.